src/models/flinkRelation.ts

```diff
@@ -366,13 +372,9 @@ export class FlinkRelation {
       tooltip.addField("Watermarked", "No");
     }
 
-    // List visible columns
-    const visible = this.visibleColumns;
-    if (visible.length === 0) {
-      tooltip.addField("Visible Columns", "None");
-    } else {
-      const rendered = visible.map((c) => c.tooltipLine());
-      tooltip.addField("Visible Columns", rendered.join("\n"));
+    // If has a view definition, show it here for the time being.
+    if (this.type === FlinkRelationType.View && this.viewDefinition) {
+      tooltip.addField("View Definition", `\`\`\`\n${this.viewDefinition}\n\`\`\``);
```

Comment:
More a note-to-self, but we'll want to update this when working on https://github.com/confluentinc/vscode/issues/1600 (and maybe https://github.com/confluentinc/vscode/issues/461) as well.